internal/cmd/cmd.go

```diff
@@ -172,7 +172,11 @@ func defaultAPIClient() (*api.Client, error) {
 func apiClient(host string, accessKey string, skipTLSVerify bool) (*api.Client, error) {
 	u, err := urlx.Parse(host)
 	if err != nil {
-		return nil, err
+		return nil, UserFacingError{
+			UserFacingMessage: fmt.Sprintf("Could not reach %s. Please check server hostname.", host),
```

Comment:
I wanted a general statement saying "something is wrong with your hostname, whether it is during parsing or connecting" (and then print the actual error). Do you have any suggestions for the wording since the current one could be misleading? 

since to the user, it is a bad hostname, whether it be infra or the server rejecting it